numpy/core/tests/test_multiarray.py

```diff
@@ -5457,6 +5457,47 @@ def test_relaxed_strides(self):
             shape, strides = get_buffer_info(arr, ['C_CONTIGUOUS'])
             assert_(strides[-1] == 8)
 
+    # test views of different dtype sizes. These tests will break when
+    # relaxed strides are enabled
+    @dec.skipif(np.ones(1).strides[0] == np.iinfo(np.intp).max)
```

Comment:
This is probably unsafe as the strides are reset to the usual for release. Safer would be to check if `ones((2,1), order='F')` is c_contiguous (as well as f_contiguous).